site/content/blog/2021-11-01-whats-new-in-svelte-november-2021.md

```diff
@@ -0,0 +1,85 @@
+---
+title: What's new in Svelte: November 2021
+description: Over 5000 stars to light up the showcase
+author: Daniel Sandoval
+authorURL: https://desandoval.net
+---
+
+With SvelteKit crossing both the [80% complete mark](https://github.com/sveltejs/kit/milestone/2) _and_ over [5000 stars](https://github.com/sveltejs/kit) on GitHub, now is a great time to try it out! Many in the community have, making for quite a large showcase this month...
+
+Also, don't miss out on [Svelte Summit](https://sveltesummit.com/) on November 20th - featuring speakers from around the world. Keep an eye out for a watch party in your area 👀
+
+Now onto what's new!
+
+## New in Svelte and SvelteKit
+
+- A new compiler option, `enableSourcemap`, provides more control over the compiler output for JS and CSS sourcemaps (**3.44.0**). With this new feature, SvelteKit and the Vite Svelte plugin can now properly handle environment variables (See [sveltejs/kit#720](https://github.com/sveltejs/kit/issues/720) and [sveltejs/vite-plugin-svelte#201](https://github.com/sveltejs/vite-plugin-svelte/pull/201))
+- The Svelte Language Tools now support configuring the VS Code CSS Language Service settings ([#1219](https://github.com/sveltejs/language-tools/issues/1219))
+- SvelteKit will only route endpoints on the client, unless marked as `rel="external"` - reducing the size of the client JS and making it easier to refactor the router in the future ([2656](https://github.com/sveltejs/kit/pull/2656))
+- SvelteKit no longer supports Node 12 ([2604](https://github.com/sveltejs/kit/pull/2604))
+
```

Comment:
- SvelteKit was upgraded from Vite 2.6.0 to Vite 2.6.12 fixing an issue where Vite would corrupt the Svelte runtime (https://github.com/vitejs/vite/issues/4306). It also included two fixes from the SvelteKit team to avoid or make diagnosing Vite issues in SvelteKit templates easier (https://github.com/vitejs/vite/pull/5192 and https://github.com/vitejs/vite/pull/5193). Vite 2.7 is currently available in beta with additional fixes for SSR